package.json

```diff
@@ -118,7 +118,7 @@
     "selenium-server": "^2.53.1",
     "serialize-javascript": "^1.3.0",
     "shelljs": "^0.7.8",
-    "typescript": "^2.3.4",
+    "typescript": "2.5.0-dev.20170615",
```

Comment:
It depended on some experimental changes but afaik they landed in 2.4, didn't they @DanielRosenwasser?